modules/objdetect/src/aruco/apriltag/zmaxheap.cpp

```diff
@@ -0,0 +1,207 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2013-2016, The Regents of The University of Michigan.
+//
+// This software was developed in the APRIL Robotics Lab under the
+// direction of Edwin Olson, ebolson@umich.edu. This software may be
+// available under alternative licensing terms; contact the address above.
+//
+// The views and conclusions contained in the software and documentation are those
+// of the authors and should not be interpreted as representing official policies,
+// either expressed or implied, of the Regents of The University of Michigan.
+
+#include <opencv2/core.hpp>
```

Comment:
Wrong precomp.hpp usage.

All files of opencv modules must follow these rules:
- .cpp file: one and only one (1) usage of precomp.hpp include on the first place (2).
- .hpp file: can't use precomp.hpp at all

Reply:
fixed, sorry part of cpp previously missed

Reply:
precomp.hpp was added to all objdetect/aruco cpp files and removed from all objdetect/aruco hpp files